mrtt-ui/src/App.js

```diff
@@ -22,7 +22,7 @@ import SiteForm from './views/SiteForm'
 import SiteQuestionsOverview from './views/SiteQuestionsOverview/SiteQuestionsOverview'
 import Sites from './views/Sites'
 import themeMui from './styles/themeMui'
-import CostsForm from './views/CostsForm'
+import CostsForm from './components/CostsForm'
 
```

Comment:
I think we have divergent approaches to views vs components. In a perfect world with perfect timelines this would be worth a discussion. Not a blocker, just a comment.

Reply:
I noticed you added this one into views, however, we still have other forms in components so I was confused as to which way we wanted to go. I do see them as components, however, I agree it would be worth having a discussion over. As long as we keep it consistent!